iwyu_tool.py

```diff
@@ -203,6 +208,8 @@ def custom_help():
     parser.add_argument('source', nargs='*',
                         help='Zero or more source files to run IWYU on. '
                         'Defaults to all in compilation database.')
+    parser.add_argument('-i', '--ignore', nargs='+',
```

Comment:
I think it just makes it require at least 1 argument following if `-i` or `--ignore` is specified.